ledger/compact_merkle_tree.py

```diff
@@ -261,10 +261,24 @@ def leafCount(self) -> int:
     def nodeCount(self) -> int:
         return self.hashStore.nodeCount
 
-    def verifyConsistency(self, expectedLeafCount = -1) -> bool:
-        if expectedLeafCount > 0 and expectedLeafCount != self.leafCount:
+    @staticmethod
+    def get_expected_node_count(leaf_count):
```

Comment:
Well we store full subtree roots in CompactMerkleTree, i am not sure whether we will need to store them in case of other kinds of trees, but we don't have others as of now